aiven/client/cli.py

```diff
@@ -138,7 +138,6 @@ class ClientFactory(Protocol):  # pylint: disable=too-few-public-methods
         def __call__(self, base_url: str, show_http: bool, request_timeout: Optional[int]) -> client.AivenClient:
             ...
 
-
```

Comment:
Should that change has been reverted?

Reply:
I believe it's black version update which is guilty for this change. Otherwise linters won't pass.